src/uikit.cs

```diff
@@ -23827,4 +24636,743 @@ interface UICellAccessoryDetail {
+	}
+
+	interface IUITextSearching {}
+
+	[NoWatch, NoTV, iOS (16,0), MacCatalyst (16,0)]
+	[Protocol]
+	interface UITextSearching
+	{
+		[Abstract]
+		[NullAllowed, Export ("selectedTextRange")]
+		UITextRange SelectedTextRange { get; }
+
+		[Abstract]
+		[Export ("compareFoundRange:toRange:inDocument:")]
+		NSComparisonResult CompareFoundRange (UITextRange foundRange, UITextRange toRange, [NullAllowed] INSCopying document);
+
+		[Abstract]
+		[Export ("performTextSearchWithQueryString:usingOptions:resultAggregator:")]
+		void PerformTextSearch (string @string, UITextSearchOptions options, IUITextSearchAggregator aggregator);
+
+		[Abstract]
+		[Export ("decorateFoundTextRange:inDocument:usingStyle:")]
+		void DecorateFoundTextRange (UITextRange range, [NullAllowed] INSCopying document, UITextSearchFoundTextStyle style);
+
+		[Abstract]
+		[Export ("clearAllDecoratedFoundText")]
+		void ClearAllDecoratedFoundText ();
+
+		[Export ("supportsTextReplacement")]
+		bool SupportsTextReplacement { get; }
+
+		[Export ("shouldReplaceFoundTextInRange:inDocument:withText:")]
+		bool ShouldReplaceFoundText (UITextRange range, [NullAllowed] INSCopying document, string replacementText);
+
+		[Export ("replaceFoundTextInRange:inDocument:withText:")]
+		void ReplaceFoundText (UITextRange range, [NullAllowed] INSCopying document, string replacementText);
+
+		[Export ("replaceAllOccurrencesOfQueryString:usingOptions:withText:")]
+		void ReplaceAllOccurrences (string queryString, UITextSearchOptions options, string replacementText);
+
+		[Export ("willHighlightFoundTextRange:inDocument:")]
+		void WillHighlightFoundTextRange (UITextRange range, [NullAllowed] INSCopying document);
+
+		[Export ("scrollRangeToVisible:inDocument:")]
+		void ScrollRangeToVisible (UITextRange range, [NullAllowed] INSCopying document);
+
+		[NullAllowed, Export ("selectedTextSearchDocument")]
+		INSCopying SelectedTextSearchDocument { get; }
+
+		[Export ("compareOrderFromDocument:toDocument:")]
+		NSComparisonResult CompareOrder (INSCopying fromDocument, INSCopying toDocument);
+	}
+
+	[NoWatch, NoTV, iOS (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIEditMenuInteraction : UIInteraction
+	{
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		IUIEditMenuInteractionDelegate Delegate { get; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; }
+
+		[Export ("initWithDelegate:")]
+		NativeHandle Constructor ([NullAllowed] IUIEditMenuInteractionDelegate @delegate);
+
+		[Export ("presentEditMenuWithConfiguration:")]
+		void PresentEditMenu (UIEditMenuConfiguration configuration);
+
+		[Export ("dismissMenu")]
+		void DismissMenu ();
+
+		[Export ("reloadVisibleMenu")]
+		void ReloadVisibleMenu ();
+
+		[Export ("updateVisibleMenuPositionAnimated:")]
+		void UpdateVisibleMenuPositionAnimated (bool animated);
+
+		[Export ("locationInView:")]
+		CGPoint LocationInView ([NullAllowed] UIView view);
```

Comment:
```suggestion
		CGPoint GetLocationInView ([NullAllowed] UIView view);
```
Or maybe "GetLocation"